textile/chat-features.textile

```diff
@@ -104,121 +104,146 @@ The status of any given Chat Room is the combination of the states of all of its
+There are three room lifecycle operations: @ATTACH@, @DETACH@, @RELEASE@.
 
-There are four room lifecycle operations: @ATTACH@, @DETACH@, @RELEASE@ and @RETRY@. @RETRY@ is an internal-only operation. These operations are intended to be atomic and not interfere with each other, in order to preserve chat room integrity. How this is achieved is described in further specification items.
-
-Discontinuities in Realtime connections happen - whereby continuity of message delivery is disrupted. Therefore each feature may also experience discontinuity events - where the user may need to take some action to restore continuity in their application. In Chat, we explicitly tell the user when there's a discontinuity, rather than require them to implement the monitoring themselves. As such are events are normally associated with something going wrong, we only want to tell them that a discontinuity happened when everything has fixed itself. Therefore, when a discontinuity is noticed by us, we might not notify the user immediately, instead preferring to hold the event in pending until the right time. For more information on how to handle discontinuities, consult the room lifecycle specification points.
-
-In the same sense as discontinuities, sometimes connections drop momentarily - due to connection balancing by the Realtime system or simply bad internet connections. In Chat we try to avoid broadcasting these transient disconnects to users, so long as they don't affect continuity. Therefore, we tend to handle disconnections or detachments optimistically at first - hoping that they will resolve themselves, and only informing the user when it is clear that it may take longer than anticipated.
+Discontinuities in Realtime connections can happen - whereby continuity of message delivery is disrupted. Therefore each feature may also experience discontinuity events - where the user may need to take some action to restore continuity in their application. In Chat, we explicitly tell the user when there's a discontinuity, rather than require them to implement the monitoring themselves.
 
 h4(#rooms-lifecycle-operations). Room Lifecycle Operations
 
+* @(CHA-RL13)@ The room lifecycle shall maintain a @isFirstAttach@ flag, which represents whether or not the next @ATTACHED@ channel state change is the first one received (i.e. the product of a call to @ATTACH@). It defaults to @false@. Subsequent specification items will explain how this flag is manipulated.
```

Comment:
From what I can tell, this flag starts off as `false` and only ever gets set to `false`?

Reply:
This should be clarified

Reply:
Looks like this is fixed now, right?

Reply:
Yes :)